packages/extension/src/ui/features/actions/AddTokenScreen.tsx

```diff
@@ -51,7 +51,7 @@ const ButtonSpacer = styled.div`
 `
 
 function isValidUrl(url: string | undefined): boolean {
-  return !!url && yup.string().url().isValidSync(url)
+  return !url || !!url && yup.string().url().isValidSync(url)
```

Comment:
Question: Is the purpose of this change to avoid validating the URL?

Reply:
Yes, URL is optional, not mandatory. So if URL is not specified we should still be able to add the token.